public/js/validations.js

```diff
@@ -14,4 +14,41 @@ function isTime(time) {
     } else {
         return false
     }
+}
+
+function isPhoneNumber(phone) {
+    let regEx = /^(([+][(]?[0-9]{1,3}[)]?)|([(]?[0-9]{4}[)]?))\s*[)]?[-\s\.]?[(]?[0-9]{1,3}[)]?([-\s\.]?[0-9]{3})([-\s\.]?[0-9]{3,4})$/gm
```

Comment:
after the country code, if the first digit is 0 then you should omit it, so for example it goes:
+44(0)7100123345
Does this kind of filtering for display be done in this branch do you think?